code/modules/mob/living/simple_animal/bot/mulebot.dm

```diff
@@ -319,13 +319,13 @@
 /mob/living/simple_animal/bot/mulebot/proc/buzz(type)
 	switch(type)
 		if(SIGH)
-			audible_message("[src] makes a sighing buzz.", "<span class='hear'>You hear an electronic buzzing sound.</span>")
+			audible_message("<span class='hear'>[src] makes a sighing buzz.</span>", "<span class='hear'>You hear an electronic buzzing sound.</span>")
 			playsound(loc, 'sound/machines/buzz-sigh.ogg', 50, FALSE)
 		if(ANNOYED)
-			audible_message("[src] makes an annoyed buzzing sound.", "<span class='hear'>You hear an electronic buzzing sound.</span>")
+			audible_message("<span class='hear'>[src] makes an annoyed buzzing sound.</span>", "<span class='hear'>You hear an electronic buzzing sound.</span>")
 			playsound(loc, 'sound/machines/buzz-two.ogg', 50, FALSE)
 		if(DELIGHT)
-			audible_message("[src] makes a delighted ping!", "<span class='hear'>You hear a ping.</span>")
+			audible_message("<span class='hear'>[src] makes a delighted ping!</span>", "<span class='hear'>You hear a ping.</span>")
```

Comment:
Not your fault but the second argument in audible message is deaf_message meaning that people that can see but can't hear receive it.

Reply:
Alright, I'll remove those and leave it with just the first argument.